common/build.gradle

```diff
@@ -159,6 +160,10 @@ dependencies {
     //For Executive Order work
     implementation "com.yubico.yubikit:android:$rootProject.ext.yubikitAndroidVersion"
     implementation "com.yubico.yubikit:piv:$rootProject.ext.yubikitPivVersion"
+    //For passkeys
+    implementation "androidx.credentials:credentials:1.2.0-beta01"
```

Comment:
are we planning to GA this feature with beta?

Reply:
We'll GA with the latest tested version that CredMan has available (at the time of public preview/GA). They seem to be updating on a monthly cadence: https://developer.android.com/jetpack/androidx/releases/credentials#version_12_2.